tests/unitTests/unitTest.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<svFSIFile version="0.1">
+
+<GeneralSimulationParameters>
+  <Continue_previous_simulation> 0 </Continue_previous_simulation>
+  <Number_of_spatial_dimensions> 3 </Number_of_spatial_dimensions> 
+  <Number_of_time_steps> 20 </Number_of_time_steps>
+  <Time_step_size> 0.01 </Time_step_size> 
+  <Spectral_radius_of_infinite_time_step> 0.50 </Spectral_radius_of_infinite_time_step> 
+  <Searched_file_name_to_trigger_stop> STOP_SIM </Searched_file_name_to_trigger_stop> 
+
+  <Save_results_to_VTK_format> 1 </Save_results_to_VTK_format> 
+  <Name_prefix_of_saved_VTK_files> result </Name_prefix_of_saved_VTK_files> 
+  <Increment_in_saving_VTK_files> 1 </Increment_in_saving_VTK_files> 
+  <Start_saving_after_time_step> 1 </Start_saving_after_time_step> 
+
+  <Increment_in_saving_restart_files> 2 </Increment_in_saving_restart_files> 
+  <Convert_BIN_to_VTK_format> 0 </Convert_BIN_to_VTK_format> 
+
+  <Verbose> 1 </Verbose> 
+  <Warning> 0 </Warning> 
+  <Debug> 0 </Debug> 
+
+</GeneralSimulationParameters>
+
+
+<Add_mesh name="msh" > 
+
+  <Mesh_file_path>  ../../../../tests/unitTests/mesh/unitCube/unit_cube_volume.vtu </Mesh_file_path>
+
+  <Add_face name="Z0">
+      <Face_file_path>  ../../../../tests/unitTests/mesh/unitCube/mesh_surfaces/Z0.vtp </Face_file_path>
+  </Add_face>
+
+  <Add_face name="Z1">
+      <Face_file_path>  ../../../../tests/unitTests/mesh/unitCube/mesh_surfaces/Z1.vtp </Face_file_path>
+  </Add_face>
+
+  <Add_face name="Y0">
+      <Face_file_path>  ../../../../tests/unitTests/mesh/unitCube/mesh_surfaces/Y0.vtp </Face_file_path>
+  </Add_face>
+
+  <Add_face name="Y1">
+      <Face_file_path>  ../../../../tests/unitTests/mesh/unitCube/mesh_surfaces/Y1.vtp </Face_file_path>
+  </Add_face>
+
+  <Add_face name="X0">
+      <Face_file_path>  ../../../../tests/unitTests/mesh/unitCube/mesh_surfaces/X0.vtp </Face_file_path>
+  </Add_face>
+
+  <Add_face name="X1">
+      <Face_file_path>  ../../../../tests/unitTests/mesh/unitCube/mesh_surfaces/X1.vtp </Face_file_path>
+  </Add_face>
+
+  <Mesh_scale_factor> 0.001 </Mesh_scale_factor> 
```

Comment:
Would remove since it's not necessary

Reply:
I'll try to simplify the input file and remain only the necessary lines.